srunner/scenariomanager/scenarioatomics/atomic_criteria.py

```diff
@@ -1049,6 +1042,20 @@ def update(self):
 
                 self._current_index = index
 
+        # Deactivate/Activate cheching by blackboard message
+        activate = py_trees.blackboard.Blackboard().get('AC_SwitchOutsideRouteLanesTest')
```

Comment:
NO! What I was asking for is to change [this](https://github.com/carla-simulator/scenario_runner/blob/master/srunner/scenariomanager/scenarioatomics/atomic_criteria.py#L1125) conditional to also take into account that blackboard. To something like:

```py
if self._active and (self._outside_lane_active or self._wrong_lane_active):
      self._wrong_distance += new_dist
```

Additionally:

```py
active = py_trees.blackboard.Blackboard().get('AC_SwitchOutsideRouteLanesTest')
if active is not None:
    self._active = active
    py_trees.blackboard.Blackboard().set("AC_SwitchOutsideRouteLanesTest", None, overwrite=True)

self._is_outside_driving_lanes(location)
self._is_at_wrong_lane(location)

if self._active and (self._outside_lane_active or self._wrong_lane_active):
      self.test_status = "FAILURE"
```

Reply:
Get it. Done.